### PR TITLE
Add subject support for Email survey API

### PR DESCRIPTION
**Pull request by marinhero**
- Add subject support for Email survey API

### CODE REVIEW

**Review by diegoserranoa — APPROVED** [review]
Looks good!

File: source/includes/_email_survey.md [hunk 1]
```diff
@@ -6,10 +6,11 @@ curl "https://api.wootric.com/v1/email_survey" \
   -d "emails[]=john@example.com" \
   -d "emails[]=jane@example.com" \
   -d "survey_immediately=true" \
-  -d "survey_settings[custom_messages][followup_text]=Thank you!"
+  -d "survey_settings[custom_messages][followup_text]=Thank you!" \
+  -d "subject='Would you mind sharing your thoughts about our service?'"
```
Comment (diegoserranoa) [inline source/includes/_email_survey.md]: Are the single quotes needed?

**Comment by marinhero** [comment]
yes

File: source/includes/_email_survey.md (hunk 1 above)
Comment (marinhero) [reply source/includes/_email_survey.md]: No, I'll remove them !

**Comment by marinhero** [comment]
Updated!

**Pull request merged**